src/mame/drivers/namcos12.cpp

```diff
@@ -1736,6 +1758,47 @@ MACHINE_CONFIG_START(namcos12_state::coh700)
 	//MCFG_SOUND_ROUTE(3, "rspeaker", 1.00)
 MACHINE_CONFIG_END
 
+MACHINE_CONFIG_START(namcos12_state::coh716)
```

Comment:
Could you make a base machine configuration with the common parts, and only put what's different in the `coh700` and `coh716` configurations?  It's not clear at all what he actual differences are when it's like this.